### ISSUE TITLE
(possible false alarm) Windows Defender reports pass-winmenu.exe as a trojan

### ISSUE THREAD

**Issue by brandon-arnold** [Baggykiin/pass-winmenu#44]
I apologize for any undue alarm this brings, but Windows Defender just reported this `.exe` as a Trojan. In the event this is a false positive or a result of a trojan that is on my machine unrelated to pass-winmenu, I will try to update here. At any rate within moments of unzipping the current latest release, Windows Defender quarantined the `.exe` and I found the following report in the Windows Defender details:

![2019-03-10](https://user-images.githubusercontent.com/7827691/54094192-08d23180-436d-11e9-8bf0-e5a1212111e8.png)

**Comment by willpower232** [comment]
Have you updated the virus definitions? I use this all the time and get no troubles. Tried a fresh download too but can't replicate what you're getting.

**Comment by brandon-arnold** [comment]
> Have you updated the virus definitions? I use this all the time and get no troubles. Tried a fresh download too but can't replicate what you're getting.

It is VERY POSSIBLE the virus definitions were out of date. I had booted the laptop down before my move to Seattle and it was ~1 month behind on updates when I booted it up yesterday. I was using (and updating) `pass-winmenu` before that was complete. I will check again now that updates are finished when I'm back home tonight and, provided successful results, I'll close this for sure.

Thanks!
